docs/tools/firmware.sh

```diff
@@ -215,7 +233,7 @@ case "$os" in
 		echo "Installing Wi-Fi and Bluetooth firmware"
 		sudo mount ${verbose} /dev/nvme0n1p1 $mountpoint
 		sudo tar --warning=no-unknown-keyword ${verbose} -xC $workdir -f $mountpoint/firmware.tar.gz
-		sudo python3 "$0" $workdir $workdir/firmware-renamed.tar ${verbose}
+		sudo python3 $0 $workdir $workdir/firmware-renamed.tar ${verbose}
```

Comment:
Word splitting can cause problems here, leave it in double quotes

Reply:
My bad, that change was unintentional.